app/services/StubIdentityService.scala

```diff
@@ -0,0 +1,23 @@
+package services
+
+import cats.data.EitherT
+import com.gu.identity.play.{IdMinimalUser, IdUser, PrivateFields, PublicFields}
+import monitoring.SafeLogger
+import play.api.mvc.RequestHeader
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class StubIdentityService extends IdentityService {
+  def getUser(user: IdMinimalUser)(implicit req: RequestHeader, ec: ExecutionContext): EitherT[Future, String, IdUser] = {
+    val privateFields = PrivateFields(firstName = Some("Frosty"), secondName = Some("The Snowman"))
+    val stubTestUser: IdUser = IdUser("123456", "nonsense@gu.com", PublicFields(None), Some(privateFields), None)
+
+    SafeLogger.info(s"Stubbed identity service active. Returning test names $privateFields")
+    EitherT(Future.successful(Either.cond(true, stubTestUser, "This stub should always give a right")))
```

Comment:
I haven't tested this, but it does compile:
`import cats.implicits._`
`EitherT.rightT[Future, String](stubTestUser)`

Reply:
Yep, that works. I'm glad to be able to remove the pointless left that should never happen. Thanks.